README.md

```diff
@@ -7,28 +7,39 @@ to lower the barrier to participating in release votes.
 ## Work in Progress
 This is very early days.. you'll need to also manually verify to double-check. Ex we have a [wiki](https://cwiki.apache.org/confluence/display/ZIPKIN/Verifying+a+Source+Release).
 
-## How to
-This technical steps will change over time, likely ending up a docker image. So, this will drift!
+## Just run it!
 
+This requires Docker to be installed on your machine, but nothing else.
 Main idea is you need to look at release VOTE thread and enter a few things:
 
-Ex.
 ```bash
-$ cd src
+./check.sh brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635
+```
+
+## The hard way
+
+For hacking on the script, or running it without Docker, you'll need some system-level dependencies.
+A non-complete list (the script will fail when it hits a missing one anyway):
+
+* wget
+* gpg
+* sha256sum
+* git
+* maven
+
+```bash
 $ virtualenv --python=python3 venv
 $ . venv/bin/activate
 $ pip install -r requirements.txt
 ### and now actually run it!
-$ python main.py brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635
+$ python src/main.py brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635
 ```
 
 ## Hacking
 
 Install linter, type-checker, and code formatter; run them before committing:
 
 ```bash
-$ cd src
-$ . venv/bin/activate
```

Comment:
they still need to activate the virtualenv

Reply:
Well. If we include that, then also setting up the virtualenv needs to be here. I'll do that for now just to make things completely clear. All this will go away with a proper `Makefile` pretty soon.